model/jpa/src/main/resources/META-INF/jpa-changelog-26.0.0.xml

```diff
@@ -45,4 +45,14 @@
         </createIndex>
     </changeSet>
 
+    <changeSet author="keycloak" id="31296-persist-revoked-access-tokens">
+        <createTable tableName="REVOKED_TOKEN">
+            <column name="ID" type="VARCHAR(255)">
+                <constraints nullable="false" />
+            </column>
+            <column name="EXPIRE" type="INT"/>
```

Comment:
If I understand correctly this uses 4 bytes integer. Should we use bigint instead to avoid the year 2038 problem?

Reply:
There are a lot of other places where INT is used for time in seconds. Still, we can move to 8 bytes here. Part of the next change.